test/network_test.ml

```diff
@@ -12,15 +12,9 @@
  * GNU Lesser General Public License for more details.
  *)
 
-open OUnit
-
-let base_suite =
-  "base_suite" >:::
-  [
-    Network_test_lacp_properties.suite;
-    Test_jsonrpc_client.suite;
-  ]
-
-let _ = 
-  Coverage.init "network_test";
```

Comment:
Was this not doing anything?

Reply:
@edwintorok says it may not actually work...

Reply:
i shouldn't have removed it, my bad

it sets the `BISECT_FILE` env variable - i will check that it is still behaving reasonably